src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -84,6 +87,19 @@ public function getImpersonatePath(string $identifier): string
         return $this->impersonateUrlGenerator->generateImpersonationPath($identifier);
     }
 
+    public function userIsGranted(UserInterface $user, mixed $role, mixed $object = null, ?string $field = null): bool
+    {
+        if (null === $this->userSecurityChecker) {
+            return false;
```

Comment:
should this return false or throw an exception saying that this function cannot be used when a UserAuthorizationCheckerInterface is not provided ?

Reply:
IMO, I'd rather throw a meaningful exception here as it's more convenient for the developer to understand what's going on.

Reply:
IMO, I'd rather throw a meaningful exception here as it's more convenient for the developer to understand what's going on.